src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating(){ return false; }
```

Comment:
This is okay, but do consider the semantics of having isMutating default to false. Another way to not attach a default result to a parent class could be using Interfaces.